packages/console/src/pages/JwtClaims/MonacoCodeEditor/index.module.scss

```diff
@@ -0,0 +1,56 @@
+@use '@/scss/underscore' as _;
+@use '@/scss/code-editor' as codeEditor;
+
+.codeEditor {
+  @include codeEditor.color;
+  @include codeEditor.font;
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  border-radius: 8px;
+  background-color: var(--color-code-bg);
+
+
+  header {
+    padding: _.unit(4);
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+
+    .tabList {
+      display: flex;
+      gap: _.unit(2);
+
+      .tab {
+        padding: _.unit(1.5) _.unit(3);
+        font: 500 14px/20px 'Roboto Mono', monospace;
```

Comment:
nit: Can we use `var(--font-label-2)` and then override the font-family?

Reply:
updated